test/spec/modules/magniteAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,1864 @@
+        'bidderCode': 'rubicon',
+        'bids': [
+          {
+            'bidder': 'rubicon',
+            'params': {
+              'accountId': 1001,
+              'siteId': 267318,
+              'zoneId': 1861698,
+            },
+            'adUnitCode': 'box',
+            'transactionId': '7b10a106-89ea-4e19-bc51-9b2e970fc42a',
+            'bidId': '23fcd8cf4bf0d7',
+            'src': 'client',
+            'startTime': 1658868383748
+          }
+        ],
+        'refererInfo': {
+          'page': 'http://a-test-domain.com:8000/test_pages/sanity/TEMP/prebidTest.html?pbjs_debug=true',
+        },
+      }
+    ],
+    'timeout': 3000,
+    'config': {
+      'accountId': 1001,
+      'endpoint': 'https://pba-event-service-alb-dev.use1.fanops.net/event'
+    }
+  },
+  BID_REQUESTED: {
+    'bidderCode': 'rubicon',
+    'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+    'bids': [
+      {
+        'bidder': 'rubicon',
+        'params': {
+          'accountId': 1001,
+          'siteId': 267318,
+          'zoneId': 1861698,
+        },
+        'adUnitCode': 'box',
+        'bidId': '23fcd8cf4bf0d7',
+        'src': 'client',
+      }
+    ]
+  },
+  BID_RESPONSE: {
+    'bidderCode': 'rubicon',
+    'width': 300,
+    'height': 250,
+    'adId': '3c0b59947ced11',
+    'requestId': '23fcd8cf4bf0d7',
+    'transactionId': '7b10a106-89ea-4e19-bc51-9b2e970fc42a',
+    'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+    'mediaType': 'banner',
+    'source': 'client',
+    'currency': 'USD',
+    'creativeId': '4954828',
+    'cpm': 3.4,
+    'ttl': 300,
+    'netRevenue': true,
+    'ad': '<html></html>',
+    'bidder': 'rubicon',
+    'adUnitCode': 'box',
+    'timeToRespond': 271,
+    'size': '300x250',
+    'status': 'rendered',
+    getStatusCode: () => 1,
+  },
+  AUCTION_END: {
+    'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+    'auctionEnd': 1658868384019,
+  },
+  BIDDER_DONE: {
+    'bidderCode': 'rubicon',
+    'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+    'bids': [
+      {
+        'bidder': 'rubicon',
+        'adUnitCode': 'box',
+        'transactionId': '7b10a106-89ea-4e19-bc51-9b2e970fc42a',
+        'bidId': '23fcd8cf4bf0d7',
+        'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+        'src': 'client',
+      }
+    ]
+  },
+  BID_WON: {
+    'bidderCode': 'rubicon',
+    'bidId': '23fcd8cf4bf0d7',
+    'adId': '3c0b59947ced11',
+    'requestId': '23fcd8cf4bf0d7',
+    'transactionId': '7b10a106-89ea-4e19-bc51-9b2e970fc42a',
+    'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+    'mediaType': 'banner',
+    'currency': 'USD',
+    'cpm': 3.4,
+    'ttl': 300,
+    'bidder': 'rubicon',
+    'adUnitCode': 'box',
+    'status': 'rendered',
+  }
+}
+
+const ANALYTICS_MESSAGE = {
+  'channel': 'web',
+  'integration': 'pbjs',
+  'referrerUri': 'http://a-test-domain.com:8000/test_pages/sanity/TEMP/prebidTest.html?pbjs_debug=true',
+  'version': '7.8.0-pre',
```

Comment:
Almost all tests are failing and will continue to fail because current master is at `7.13.0-pre`... So probably it's worth to remove this field from the comparison

Reply:
duh 🤦‍♂️